backend-diagram-converter/core/src/test/java/org/camunda/community/converter/RetryTimeCycleConverterTest.java

```diff
@@ -0,0 +1,18 @@
+package org.camunda.community.converter;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class RetryTimeCycleConverterTest {
+
+  @ParameterizedTest
+  @CsvSource(
+      value = {"R5/PT1S; 5", "PT3S, PT4S, PT5S; 3", "R5/PT1S, PT10M; 6"},
```

Comment:
> "R5/PT1S, PT10M; 6"

This combination is not possible: https://camunda.slack.com/archives/C80C0EC79/p1668083409447289

Reply:
Good to know, I was not sure. So the function can do more than it could.

Reply:
Copy the original implementation? We should not add functionality that is not implemented in the Camunda Platform 7.

Reply:
Removed functionality. Instead, focus on robust implementation.